test/e2e/autoscale_test.go

```diff
@@ -118,45 +117,23 @@ func generateTraffic(clients *test.Clients, logger *logging.BaseLogger, concurre
 	return nil
 }
 
-func getAutoscalerConfigMap(clients *test.Clients) (*v1.ConfigMap, error) {
-	return test.GetConfigMap(clients.KubeClient).Get("config-autoscaler", metav1.GetOptions{})
-}
-
-func setScaleToZeroThreshold(clients *test.Clients, threshold string, gracePeriod string) error {
-	configMap, err := getAutoscalerConfigMap(clients)
-	if err != nil {
-		return err
-	}
-	configMap.Data["scale-to-zero-threshold"] = threshold
-	configMap.Data["scale-to-zero-grace-period"] = gracePeriod
-	_, err = test.GetConfigMap(clients.KubeClient).Update(configMap)
-	return err
-}
-
 func setup(t *testing.T, logger *logging.BaseLogger) *test.Clients {
 	clients := Setup(t)
 
-	configMap, err := getAutoscalerConfigMap(clients)
+	configMap, err := test.GetConfigMap(clients.KubeClient).Get("config-autoscaler", metav1.GetOptions{})
 	if err != nil {
-		logger.Infof("Unable to retrieve the autoscale configMap. Assuming a ScaleToZero value of '5m'. %v", err)
-		initialScaleToZeroThreshold = "5m"
-		initialScaleToZeroGracePeriod = "2m"
-	} else {
-		initialScaleToZeroThreshold = configMap.Data["scale-to-zero-threshold"]
-		initialScaleToZeroGracePeriod = configMap.Data["scale-to-zero-grace-period"]
+		t.Fatalf("Unable to get autoscaler config map: %v", err)
 	}
-
-	err = setScaleToZeroThreshold(clients, "1m", "30s")
+	threshold, err := time.ParseDuration(configMap.Data["scale-to-zero-threshold"])
 	if err != nil {
-		t.Fatalf(`Unable to set ScaleToZeroThreshold to '1m'. This will
-		          cause the test to time out. Failing fast instead. %v`, err)
+		t.Fatalf("Unable to parse scale-to-zero-threshold as duration: %v", err)
 	}
+	scaleToZeroThreshold = threshold
```

Comment:
[nit] No need of another var. can be 
`scaleToZeroThreshold, err := time.ParseDuration(configMap.Data["scale-to-zero-threshold"])`

Reply:
Oh yeah, good catch.  Removed.